Seamless-GTSupport/SeamlessRemoteClassCompiler.class.st

```diff
@@ -33,8 +33,12 @@ SeamlessRemoteClassCompiler >> evaluate [
 		ifTrue: [ itsSelectionString ]
 		ifFalse: [ source ].
 	self source: selectedSource.
-	doItMethod := self parse methodNode generateWithSource.
 	
+	"Keeping compability with Pharo 7"
+	(OpalCompiler canUnderstand: #transformDoit) 
+		ifTrue: [ self parse. doItMethod := self transformDoit methodNode generateWithSource ] 
```

Comment:
Is #transformDoIt required for Pharo 8?

Reply:
Yes, the #parse was changed

![image](https://user-images.githubusercontent.com/39618015/82554078-885cb280-9b65-11ea-87be-aaf2574b9141.png)

![image](https://user-images.githubusercontent.com/39618015/82554122-9dd1dc80-9b65-11ea-8a5c-7a5ef4ea8c51.png)